contributing_to_docs/doc_guidelines.adoc

```diff
@@ -92,6 +92,23 @@ Try to shorten the file name as much as possible _without_ abbreviating importan
 
 If you create a directory with a multiple-word name, separate each word with an underscore, for example `backup_and_restore`. Do not create a top-level directory in the repository without checking with the docs team. In the main OpenShift docs, you can create one level of subdirectories. In the docs for features that are designed to be used with OpenShift, such as Service Mesh and OpenShift virtualization, you can create two levels of subdirectories.
 
+[NOTE]
+====
+When creating a new directory or subdirectory, you must create two symbolic links in it:
+
+* An `images` symbolic link to the top-level `images/` directory
+* A `modules` symbolic link to the top-level `modules/` directory
+
+If the directory that contains an assembly does not have the `images` symbolic link, any images in that assembly or its modules will not be included properly when building the docs.
+
+You can use the following commands to create the symbolic links:
+
+----
+$ ln -s <path/to>/images/ images
```

Comment:
@bergerhoffer Can you please provide an example of these commands? I misunderstood and when I used them to create my symbolic links, they turned out wonky. :sweat_smile:

Reply:
@ahardin-rh Would it help if I added something like this?

```
For example, if you are inside the `openshift-docs/cli_reference/openshift_cli` directory, your commands would look like this:

$ ln -s ../../images/ images
$ ln -s ../../modules/ modules
```
I was afraid to only put an example in there, because they people might not know that they have to adjust the number of levels to back up (`../`) depending on how deep you are.

I could also add how to undo a symbolic link (`unlink <link-name>`) if you think that would help too.

Let me know!

Reply:
Perfect! Thank you

Reply:
Updated per our slack convo

Reply:
Thanks for the updates! LGTM!